checkstyle-tester/projects-to-test-on.properties

```diff
@@ -52,3 +52,32 @@ guava|git|https://github.com/google/guava|v28.2||
 #nbia-dcm4che-tools|git|https://github.com/thprakash/nbia-dcm4che-tools|c3591e6f0f84827586db25abded6708e5386ef1a||
 # RequireThis usage
 #spring-integration|git|https://github.com/spring-projects/spring-integration.git|main|
+
+#apache-struts|git|https://github.com/apache/struts.git|master|**/apache-struts/**/resources/**/*
+
+# 'InputAllEscapedUnicodeCharacters' must be skipped because it is too big and slows down JXR
+#local-checkstyle|local|../../../|master|**/.ci-temp/**/*,**/resources-noncompilable/**/asttreestringprinter/**/*,**/resources-noncompilable/**/filefilters/**/*,**/resources-noncompilable/**/main/**/*,**/resources-noncompilable/**/suppressionsstringprinter/**/*,**/resources-noncompilable/**/gui/**/*,**/resources-noncompilable/**/javadocpropertiesgenerator/**/*,src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/javaparser/InputJavaParser.java,**/InputAllEscapedUnicodeCharacters.java,**/resources-noncompilable/**/javaparser/InputJavaParser.java,**/resources-noncompilable/**/checks/imports/unusedimports/InputUnusedImportsSingleWordPackage.java,**/target/test-classes/**/*
+#sevntu-checkstyle|git|https://github.com/sevntu-checkstyle/sevntu.checkstyle|master||
+
+#hibernate-orm|git|https://github.com/hibernate/hibernate-orm|4.2.19.Final|**/hibernate-orm/documentation/**/*
+
+#spotbugs|git|https://github.com/spotbugs/spotbugs|3.1.2||
+
+#spring-framework|git|https://github.com/spring-projects/spring-framework|v4.1.6.RELEASE||
+
+#Hbase|git|https://github.com/apache/hbase|1.1.0.1||
+
+#pmd|git|https://github.com/pmd/pmd|pmd_releases/6.21.0|**/pmd/pmd-java/src/test/**/*,**/pmd/cpd/files/*
```

Comment:
As an example with your changes, PMD is listed twice. We need each item to be unique or rename the first column so it is unique.

I don't see why we would have duplicate clone URLs for this issue.

Reply:
Understood. Changes Done